scripts/stop-sandbox.sh

```diff
@@ -1,17 +1,16 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
Our codebase applies some auto-formatting to bash files. I also had to change the first line to get the script to run.